src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -149,6 +148,64 @@ function DisplayLine({
     ]
   )
 }
+function DisplayDollarAndAtLineWithParenthesis({
+  line,
+  container = 'div'
+}: {
+  line: Line
+  container?: string
+}): JSX.Element {
+  const dollarAndAtLine = line as DollarAndAtLine
+  const element = `Transliteration__DollarAndAtLineWithParenthesis`
+  return React.createElement(
+    container,
+    { className: `${element}` },
+    <div className={`Transliteration__${dollarAndAtLine.type}`}>
+      {dollarAndAtLine.displayValue}
+    </div>
+  )
+}
+
+function DisplayGenericDollarAndAtLine({
```

Comment:
Avoid using the term "generic" unless working with [generics](https://www.typescriptlang.org/docs/handbook/generics.html).